deps.windows/checksums/vlc.zip.sha256

```diff
@@ -0,0 +1,14 @@
+<Objs Version="1.1.0.1" xmlns="http://schemas.microsoft.com/powershell/2004/04">
+  <Obj RefId="0">
+    <TN RefId="0">
+      <T>Microsoft.PowerShell.Commands.FileHashInfo</T>
+      <T>System.Object</T>
+    </TN>
+    <ToString>Microsoft.PowerShell.Commands.FileHashInfo</ToString>
+    <Props>
+      <S N="Algorithm">SHA256</S>
+      <S N="Hash">91F589EF69FCE51645A3ECBB215B405C98DB7B891479474EC3B5ED3B63C25E4A</S>
+      <S N="Path">windows_32-bit_build_temp\vlc.zip</S>
```

Comment:
Why 32-bit?

Reply:
I just copied another checksum file - the Path is never used and could in theory even be omitted.

Reply:
Path is actually used here:
* https://github.com/obsproject/obs-deps/blob/713379c1cd2f556ca2c759c0e139a985273c0ff7/utils.pwsh/Invoke-SafeWebRequest.ps1#L76-L80

Reply:
Yeah I guess I limited it to the filename component because of the 32bit/64bit dichotomy and I didn't want to maintain different checksums for each architecture.

Would be better to output just the filenames and not full paths then.

Reply:
Discussed off-thread, and this should work as-is, but we'll likely remove all leading path components from the Windows hash files in a separate PR after.